README.md

```diff
@@ -2,12 +2,75 @@
 
 Open-source 4G EnodeB and User emulators developed by [Software Radio Systems (SRS)](https://www.srslte.com/).
 
-## Usage
+## Installation and configuration
 
 ```bash
```

Comment:
You can remove those 3 lines

Reply:
Which 3 lines?

Reply:
The lines starting where I added the comment

Reply:
Already removed.